mpact/services.py

```diff
@@ -262,6 +262,15 @@ def schedule_messages(xlsx_file):
 
     def is_blank(val):
         return val is None or val == ''
+    
+    def is_positive_numeric(val):
+        try:
+            val=int(val)
+            if val >= 0:
+                return True
```

Comment:
I'm being petty. I know. But I can't stop looking at it.
```suggestion
            return int(val) >= 0
```